interfaces/builtin/home_test.go

```diff
@@ -123,6 +124,14 @@ func (s *HomeInterfaceSuite) TestUnexpectedSecuritySystems(c *C) {
 	c.Assert(snippet, IsNil)
 }
 
-func (s *HomeInterfaceSuite) TestAutoConnect(c *C) {
+func (s *HomeInterfaceSuite) TestAutoConnectOnClassic(c *C) {
```

Comment:
@niemeyer - Thanks for the tip! :) "Please note this must be done on both ends" -- I took that to mean also in the TestAutoConnectOnCore-- if that is not the case, please advise.